src/tests/ftest/launch.py

```diff
@@ -52,2294 +68,2973 @@
+        result = run_remote(logger, hosts, find_command(source, pattern, depth, other), timeout=900)
+        if not result.passed:
+            message = f"Error running {cart_logtest} on the {source_files} files"
+            self._fail_test(self.result.tests[-1], "Process", message)
+            return 16
+        return 0
+
+    def _remove_empty_files(self, hosts, source, pattern, depth):
+        """Remove any files with zero size.
 
+        Args:
+            hosts (NodSet): hosts on which the files are located
+            source (str): where the files are currently located
+            pattern (str): pattern used to limit which files are processed
+            depth (int): max depth for find command
 
-def stop_daos_server_service(test_file, args):
-    """Stop any daos_server.service running on the hosts running servers.
+        Returns:
+            bint: status code: 0 = success, 16 = failure
 
-    Args:
-        test_file (dict): a dictionary of the test script/yaml file
-        args (argparse.Namespace): command line arguments for this program
+        """
+        logger.debug("-" * 80)
+        logger.debug("Removing any zero-length %s files in %s on %s", pattern, source, hosts)
+        other = ["-empty", "-print", "-delete"]
+        if not run_remote(logger, hosts, find_command(source, pattern, depth, other)).passed:
+            message = f"Error removing any zero-length {os.path.join(source, pattern)} files"
+            self._fail_test(self.result.tests[-1], "Process", message)
+            return 16
+        return 0
+
+    def _compress_files(self, hosts, source, pattern, depth):
+        """Compress any files larger than 1M.
 
-    Returns:
-        int: status code: 0 = success, 512 = failure
+        Args:
+            hosts (NodSet): hosts on which the files are located
+            source (str): where the files are currently located
+            pattern (str): pattern used to limit which files are processed
+            depth (int): max depth for find command
 
-    """
-    service = "daos_server.service"
-    hosts = get_hosts_from_yaml(test_file["yaml"], args, YAML_KEYS["test_servers"])
-    log("-" * 80)
-    log("Verifying {} on {} after running '{}'".format(service, hosts, test_file["py"]))
-    return stop_service(hosts, service)
+        Returns:
+            int: status code: 0 = success, 16 = failure
 
+        """
+        logger.debug("-" * 80)
+        logger.debug(
+            "Compressing any %s files in %s on %s larger than 1M", pattern, source, hosts)
+        other = ["-size", "+1M", "-print0", "|", "sudo", "-n", "xargs", "-0", "-r0", "lbzip2", "-v"]
+        result = run_remote(logger, hosts, find_command(source, pattern, depth, other))
+        if not result.passed:
+            message = f"Error compressing {os.path.join(source, pattern)} files larger than 1M"
+            self._fail_test(self.result.tests[-1], "Process", message)
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'fail_reason' in staticmethod call